app/views/posts/show.html.erb

```diff
@@ -41,62 +43,74 @@
                   method: :delete,
                   data: { confirm: "Are you sure?" } %>
         </div>
-        <% if ((@post.account_id == current_account.id) and !(@post.closed?)) or !(@post.closed?) %>
+      <% end %>
+         <% if ((@post.account_id == current_account.id) and !(@post.closed?)) or !(@post.closed?) %>
```

Comment:
why have you used 'and' and 'or' for @post.closed?

Reply:
It is changed